apps/server/src/modules/task/controller/task-copy.controller.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { createMock } from '@golevelup/ts-jest';
+import { Test, TestingModule } from '@nestjs/testing';
+import { TaskCopyUC } from '../uc/task-copy.uc';
+import { TaskCopyController } from './task-copy.controller';
+
+describe('TaskCopyController', () => {
+	let controller: TaskCopyController;
+
+	beforeEach(async () => {
+		const module: TestingModule = await Test.createTestingModule({
+			providers: [
+				{
+					provide: TaskCopyUC,
+					useValue: createMock<TaskCopyUC>(),
+				},
+			],
+			controllers: [TaskCopyController],
+		}).compile();
+
+		controller = module.get(TaskCopyController);
+	});
+
+	it('should be defined', () => {
+		expect(controller).toBeDefined();
+	});
```

Comment:
please provide more tests here for coverage, eg spying on usecase to make sure its called, and checking that the result has the correct type (or that its equal to the result of the mapper)